docs/DeveloperGuide.md

```diff
@@ -621,6 +624,9 @@ The following sequence diagram shows how the `undo` operation works when undoing
 
 ![UndoSequenceDiagram](images/UndoSequenceDiagram1.png)
 
+**Note:** The lifeline for `RedoCommand` should
```

Comment:
Should it be `UndoCommand` instead of `RedoCommand` since it refers to the Undo Sequence Diagram?